src/main/scala/io/smartdatalake/util/misc/AclUtil.scala

```diff
@@ -48,60 +49,57 @@ private[smartdatalake] case class AclElement (aclType:String, name:String, permi
   }
 }
 
+/**
+ * Utils for setting ACL's on Hadoop Filesystems
+ *
+ * Setting ACL's in the direction hierarchy is a sensitive task.
+ * In SmartDataLake ACL's can be configured for HadoopFileDataObjects.
+ * Setting ACL's on a given path of a DataObject has to
+ * 1) overwrite ACL's on the directory and its subdirectories
+ * 2) modify ACL's on the parent directories up to a certain level configured by hdfsAclMinLevelPermissionModify
+ * Additional limitations for overwriting ACL's can be configured by hdfsAclMinLevelPermissionOverwrite
+ *
+ * By default changing ACLs is also limited to the user home directory. This is configured by hdfsAclsLimitToUserHome and hdfsAclsUserHomeLevel.
+ * It checks if path at hdfsAclsUserHomeLevel ends with username.
+ *
+ * A common structure for directories on HDFS is as follows:
+ * hdfs://nameservice1/user/app_dir/<layer>/<source_name>/<feed_name>
+ *
+ * Levels are counted as follows
+ * Root hdfs://nameservice1/ or also "/" -> 0
+ * User Dir on Hadoop hdfs://nameservice1/user -> 1
+ * User Home hdfs://nameservice1/user/app_dir -> 2
+ * Layer hdfs://nameservice1/user/app_dir/<layer> -> 3
+ * Source hdfs://nameservice1/user/app_dir/<layer>/<source_name> -> 4
+ * Feed hdfs://nameservice1/user/app_dir/<layer>/<source_name>/<feed_name> -> 5
+ * ...
+ *
+ * In this structure good limitations are: MinLevelPermissionModify=2, MinLevelPermissionOverwrite=5
```

Comment:
```suggestion
 * Utils for setting ACLs on Hadoop Filesystems
 *
 * Setting ACLs in the directory hierarchy is a sensitive task.
 * In SmartDataLake ACLs can be configured for HadoopFileDataObjects.
 * Setting ACLs on a given path of a DataObject has to
 * 1) overwrite ACLs on the directory and its subdirectories
 * 2) modify ACLs on the parent directories up to a certain level configured by hdfsAclMinLevelPermissionModify
 * Additional limitations for overwriting ACLs can be configured by hdfsAclMinLevelPermissionOverwrite
 *
 * By default, changing ACLs is also limited to the user home directory. This is configured by hdfsAclsLimitToUserHome and hdfsAclsUserHomeLevel.
 * It checks if path at hdfsAclsUserHomeLevel ends with username.
 *
 * A common structure for directories on HDFS is as follows:
 * hdfs://nameservice1/user/app_dir/<layer>/<source_name>/<feed_name>
 *
 * Levels are counted as follows
 * Root hdfs://nameservice1/ or also "/" -> 0
 * User Dir on Hadoop hdfs://nameservice1/user -> 1
 * User Home hdfs://nameservice1/user/app_dir -> 2
 * Layer hdfs://nameservice1/user/app_dir/<layer> -> 3
 * Source hdfs://nameservice1/user/app_dir/<layer>/<source_name> -> 4
 * Feed hdfs://nameservice1/user/app_dir/<layer>/<source_name>/<feed_name> -> 5
 * ...
 *
 * In this structure good limitations are: MinLevelPermissionModify=2, MinLevelPermissionOverwrite=5
```

Reply:
`dfs.nameservices` in lieu of `nameservice1`, which could be to SBB specific?